blocks/library/button/index.js

```diff
@@ -80,7 +80,7 @@ registerBlockType( 'core/button', {
 		const { url, text, title, align = 'none' } = attributes;
 
 		return (
-			<div className={ `align${ align }` }>
```

Comment:
We should take a step back and figure out a consistent way for naming these classes across all blocks. Not sure `blocks-{name}` will be enough. Maybe we should use the namespace of the block `wp-core-button`? This is something we want to get right and not change.